apps/inbound-mail/src/server/inbound-mail.service.spec.ts

```diff
@@ -63,8 +63,9 @@ describe('Inbound Mail Service', () => {
         _environmentId,
         _organizationId,
         _userId,
-      };
-      await inboundMailService.inboundParseQueueService.add(jobId, jobData, _organizationId);
+      } as any;
```

Comment:
For the test, IInboundParseDataDto contained a lot of attributes that needed to be added. for now, added casting as those attributes were not tested.